src/Client/Collections.php

```diff
@@ -53,6 +54,19 @@ public function forSubject(string ...$subjectId) : self
         return $clone;
     }
 
+    public function containing(string ...$items) : self
```

Comment:
Identifiers in general are a good candidate for Value Objects. We can have a generic one easily with a string `type` and mixed `id|identifier|...`. Projects like search would find useful to have this type to pass around, as they have lots of generic code to index all blog-articles|interviews|podcast-episodes...